onnxruntime/test/providers/cuda/reduction_functions_test.cc

```diff
@@ -265,8 +265,8 @@ TEST(ReductionFunctionsTest, GetApplicableMatrixReduction) {
       [](cudnnReduceTensorOp_t cudnn_op,
          const std::vector<int64_t>& dims, const std::vector<int64_t>& axes,
          ApplicableMatrixReduction expected_reduction,
-         const optional<int>& expected_m = nullopt,
-         const optional<int>& expected_n = nullopt) {
+         const optional<int>& expected_m = std::nullopt,
```

Comment:
>std::nullopt [](http://example.com/codeflow?start=43&length=12)

would prefer to be consistent, e.g., `nullopt`
alternatively, this can be `{}` and it won't need to change when we move to std::optional

Reply:
Done.